cambiatus/frontend
# Enhancement/analysis page

## What issue does this PR close
Closes #560 , closes #556 

## Changes Proposed ( a list of new changes introduced by this PR)
- Add alert on "My Claims" page
- Add filters + ordination + tabs ("Waiting vote" and "Analyzed") on "My Claims" page
- Add new styles to "All analysis" page
- Add tabs ("Waiting vote" and "Analyzed") to "All analysis" page 
- Tweak filters on "All analysis" page
- Fix typo in `main.css`
- Format `translations/amh-ETH.json` correctly
- Update `NeoVier/elm-review-no-functions-outside-of-modules`

## How to test ( a list of instructions on how to test this PR)

To test the "My Claims" page:
1. Go to `/profile/:user/claims` (accessible through Dashboard > My Claims)
2. Check the alert/card on the top of the page is according to designs
3. Once the claims load, check if the counts on each tab are correct
4. On the "Waiting Vote" tab:
4.1. Check if there are only claims that are still pending
4.2. Check if toggling the sorting method (asc/desc) works as expected
5. On the "Analyzed" tab:
5.1. Check if there are only claims that are not votable anymore (approved, disapproved or completed)
5.2. Check if toggling the sorting method (asc/desc) works as expected
5.3. Check if switching the filters (All, Approved, Disapproved, Completed) works as expected
5.4. Mix and match with sorting and filters and check if it works as expected

To test the "All analysis" page:
1. Go to `/dashboard/analysis` (accessible through Dashboard > Actions for analysis > All)
2. On the "Waiting Vote" tab:
2.1. Check if there are only claims that you haven't voted on yet
2.2. Check if toggling the sorting method (asc/desc) works as expected
2.3. Check if you can filter by claimer
2.4. Mix and match sorting method and filtering by claimer and it works as expected
3. On the "Analyzed" tab:
3.1. Check if there are only claims that are not votable anymore (approved, disapproved or completed)
3.2. Check if toggling the sorting method (asc/desc) works as expected
3.3. Check if switching the filters (All, Approved, Disapproved) works as expected
3.4. Mix and match with sorting and filters and check if it works as expected
